### DISCUSSION TITLE
How to re-scan document after creating correspondents and tags?

### DISCUSSION THREAD

**Discussion by beriknl** [jonaswinkler/paperless-ng#1033] [Support]
Ive create some rules to identify documents and counterparts. How do I re-run the scanning of these documents to enable classification and identify correspondents?

It works for new documents, but i would like to run it on existing.

**Comment by Pyromane** [comment]
Would be great

**Comment by torbengb** [comment]
Do you know the "document retagger" management function? It sounds like it would do what you ask:
https://paperless-ng.readthedocs.io/en/latest/administration.html#document-retagger
